src/main/javascript/src/resources/components/footer/footer.html

```diff
@@ -0,0 +1,19 @@
+<template>
+	<div class="footer">
+		<!-- <img src="/assets/images/carleton-login-logo.svg" alt="" class="logo"/>  -->
+
+		<a href="http://carleton.ca/privacy/policies/" target="_blank">Privacy Policies</a>
+	</div>
+	<div class="footer">
+		<a href="http://carleton.ca/accessibility/" target="_blank">Accessibility</a>
+
+
+	</div>
+	<div class=" footer">
+		<a href="mailto:crmsupport@carleton.ca?Subject=Support%20Request%20or%20Feedback">Support Request/Feedback</a>
+	</div>
+
+	<div class="copyright">
+		Copyright - Carleton
+	</div>
+</template>
```

Comment:
```suggestion
<template>
	<div class="footer">
		<div>
			<a href="http://carleton.ca/privacy/policies/" target="_blank">Privacy Policies</a>
		</div>
		<div>
			<a href="http://carleton.ca/accessibility/" target="_blank">Accessibility</a>
		</div>
		<div>
			<a href="mailto:crmsupport@carleton.ca?Subject=Support%20Request%20or%20Feedback">Support Request/Feedback</a>
		</div>
		<div class="copyright">
			Copyright - Carleton
		</div>
	</div>
</template>
```

Reply:
Flexbox is applied to elements that are like divs. Not spans, not anchors. Putting everything you want applied to the flexbox inside a div means each of the divs are going to get the justify-content: space-between applied to them.

Reply:
Made all the changes suggested